z390test/src/test/groovy/org/z390/test/RunRtTests.groovy

```diff
@@ -0,0 +1,101 @@
+package org.z390.test
+
+import org.junit.jupiter.api.Test
+
+class RunRtTests extends z390Test {
+    var sysmac = basePath("mac")
+    var syscpy = basePath("mac")
+    var libs = ["SYSMAC(${sysmac}+)", "SYSCPY(${syscpy}+)"]
+    // rt1 = asmlg
+    var rt1Options = ['bal', 'notiming', 'stats', *libs]
+    // rt3 = mz390
+    var rt3Options = ['noasm', 'bal', 'notiming', 'stats', *libs]
+    // rt4 = mz390
+    var rt4Options = ['noasm',  'bal', 'notiming', 'stats', *libs]
```

Comment:
The rt\RT4.BAT file does not have 'noasm' --- only has 'bal', 'notiming' and 'stats'.

Reply:
Will update this